packages/prisma/prisma/schema.prisma

```diff
@@ -7,7 +7,7 @@ generator client {
 }
 model Account {
   id                 Int       @id @default(autoincrement())
```

Comment:
I think this should also be converted into a string after all.

Reply:
Done